crates/symbolicator-js/src/symbolication.rs

```diff
@@ -105,21 +105,29 @@ async fn symbolicate_js_frame(
     // we check for a valid line (i.e. >= 1) first, as we want to avoid resolving / scraping the minified
     // file in that case. we frequently saw 0 line/col values in combination with non-js files,
     // and we want to avoid scraping a bunch of html files in that case.
-    let line = if raw_frame.lineno > 0 {
-        raw_frame.lineno
-    } else {
-        return Err(JsModuleErrorKind::InvalidLocation {
-            line: raw_frame.lineno,
-            col: raw_frame.colno,
-        });
+    let line = match raw_frame.lineno {
+        Some(lineno) if lineno > 0 => lineno,
+        Some(lineno) => {
+            return Err(JsModuleErrorKind::InvalidLocation {
+                line: lineno,
+                col: raw_frame.colno,
+            });
+        }
+        None => {
+            return Err(JsModuleErrorKind::InvalidLocation {
+                line: 0,
+                col: raw_frame.colno,
+            });
+        }
     };
 
     let col = raw_frame.colno.unwrap_or_default();
+    let abs_path = raw_frame.abs_path.clone().unwrap_or_default();
 
-    let module = lookup.get_module(&raw_frame.abs_path).await;
+    let module = lookup.get_module(&abs_path).await;
```

Comment:
Not sure what we should do here, we're fetching the module identified by `""` here. This seems wrong.

@loewenheim 👀

Reply:
I found there was an unused `JsModuleErrorKind::InvalidAbsPath` which fits here nicely. b381fe373700035de8517da699787b0019b802c0

Reply:
That seems like it should work 👍